### ISSUE TITLE
Questions groups or blocks

### ISSUE THREAD

**Issue by maximloginov** [Pierre-Sassoulas/django-survey#175]
I want to implement SF-36 Health Survey, some questions in this survey are grouped in blocks, for example, see questions 3-12 in
https://rand.org/health-care/surveys_tools/mos/36-item-short-form/survey-instrument.html
This group has a header and the same options for answers. Is it possible to implement this?

Basically, there are two questions:
1. How to make a group header? e.g "The following items are about activities ... ?"  in the example above.
2. How to arrange the answers into a table?

**Comment by Pierre-Sassoulas** [comment]
> How to make a group header? e.g "The following items are about activities ... ?" in the example above.

That would be a category to create in django admin

> How to arrange the answers into a table?

This would be a new feature.

**Comment by maximloginov** [comment]
Also, category descrition does not support html tags like `<strong>` or `<em>` to emphasize some words in question...

**Comment by maximloginov** [comment]
If I try to implement this using https://github.com/Sinkler/django-radiogrid would you include the additional dependency to django-survey? The project seems to be well-maintained and compatible different python/django versions.

**Comment by maximloginov** [comment]
btw the issue #123 may be relevant to this one, @gjelsas could you confirm?

**Comment by Pierre-Sassoulas** [comment]
> If I try to implement this using https://github.com/Sinkler/django-radiogrid would you include the additional dependency to django-survey? 

Sure, but let's make it an optional ``extra_require`` guarded with conditional in the code and documented in the readme. It's more work at first but I don't want / won't be able to deal with it if the dependency is not maintained anymore (and it seems like a small lib with less than 30 stars).

**Comment by gjelsas** [comment]
Yes, Your suggestion would maybe be a way to implement that. Thanks!

**Comment by gjelsas** [comment]
> btw the issue #123 may be relevant to this one, @gjelsas could you confirm?

Yes, Your suggestion would maybe be a way to implement that. Thanks!

**Comment by pkuppens** [comment]
@maximloginov , @Pierre-Sassoulas : I'm (also) looking to implement a set of healthcare questionnaires. This radiobutton grouping would be very useful.
So my questions:

1. @Pierre-Sassoulas is this now implemented in django-survey? (Or do I need to work around this?)
2. @maximloginov , do you have a working example that you can share/document/show the steps how you implemented it?

**Comment by Pierre-Sassoulas** [comment]
I don't think so as this issue is still opened.
